### DISCUSSION TITLE
submiting information in form

### DISCUSSION THREAD

**Discussion by rmfjd78** [h2oai/wave#541] [Q&A]
Hi 
     How can i get information inserted on a form, after submiting it.

     This is my form:
`		if hash == 'menu/Importacao':
			del q.page['Home']

			q.page['Home'] = ui.form_card(box=ui.box('content'),
										  items=[ui.text_xl(content='Importação de ficheiro'),
												 ui.combobox(name='separador',
															 label='Separador: ',
															 choices=[',', ';', ':'],
															 tooltip='Separador do ficheiro'),
												 ui.text(f'file_upload={q.args.file_upload}'),
												 ui.button(name='Submit',
														   value='submit',
														   label='Submit')])`

**Comment by lo5** [comment]
I would recommend going thru the tutorials https://h2oai.github.io/wave/docs/tutorial-counter and https://h2oai.github.io/wave/docs/tutorial-todo - they cover the basics of how to handle interactivity.

Also see https://h2oai.github.io/wave/docs/arguments.
